tools/build_langserver/langserver/analyzer.go

```diff
@@ -440,6 +440,17 @@ func (a *Analyzer) IsBuildFile(uri lsp.DocumentURI) bool {
 	return a.State.Config.IsABuildFile(base)
 }
 
+// IsBuildDefFile takes a uri path and check if it's a valid .build_defs file
```

Comment:
I'm not sure we should be doing this based on the extension? Calling them `.build_defs` is just a convention